File: app/src/main/java/com/dmi/perfectreader/render/RenderParagraph.java
package com.dmi.perfectreader.render;

import java.util.List;

public class RenderParagraph extends RenderObject {
    public RenderParagraph(float width, float height, List<RenderChild> children) {
        super(width, height, children);
    }

    @Override
    public boolean canPartiallyPainted() {
        return true;
    }
}

File: app/src/main/kotlin/com/dmi/perfectreader/render/RenderParagraph.kt
package com.dmi.perfectreader.render

import com.dmi.perfectreader.location.BookRange

class RenderParagraph(
        width: Float,
        height: Float,
        children: List<RenderChild>,
        range: BookRange
) : RenderObject(width, height, children, range) {
    override fun canPartiallyPaint() = true
}
